modules/EvseV2G/connection.cpp

```diff
@@ -985,3 +1021,51 @@ int connection_start_servers(struct v2g_context* ctx) {
 
     return 0;
 }
+int create_udp_socket(const int udp_port) {
+    // find suitable addresses
+    addrinfo hints;
+    addrinfo *result, *result_pointer;
```

Comment:
Ok, most of the EvseV2G module was written in C90 style ...

Reply:
Yes, it's a weird situation, perhaps it would still be best to use c++ style as much as possible. Here was basically following the `getaddrinfo 3` manpage which I assumed would be best practicel